### PR TITLE
feat(controller): populate pypi/docker startup envs to system setting

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
populate pypi/docker startup envs to system setting
- pypi setting in startup env is synced to system setting on startup if user doesn't set them
- system setting have higher priority than startup env
- if user delete them in system setting they will be set to empty and also have higher priority than startup env
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: server/controller/src/main/resources/application.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ spring:
       static-locations: file:/opt/starwhale.java/static/, classpath:/static/
 sw:
   version: ${SW_VERSION_CONTROLLER:0.1.0:8c82767b60686f3e2bfea9dafe8c8cce5dd34f52}
+  docker:
+    registry: ${SW_DOCKER_REGISTRY_URL:docker-registry.starwhale.cn}
```
Comment (tianweidut) [inline server/controller/src/main/resources/application.yaml:16]: use empty value as the default?  same as pypi default value.

Comment (anda-ren) [reply server/controller/src/main/resources/application.yaml]: done

**Review by tianweidut — APPROVED** [review]
LGTM

**Comment by tianweidut** [comment]
@anda-ren  update docs?

**Pull request merged**